components/message/Message.razor.cs

```diff
@@ -64,11 +66,10 @@ private MessageConfig Extend(MessageConfig config)
 
         #endregion global config
 
-        private readonly List<MessageConfig> _configs
-            = new List<MessageConfig>();
+        private readonly ConcurrentDictionary<string, MessageConfig> _configDict
```

Comment:
I think it's better to use `ConcurrentQueue` because you don't have to handle the `Order`.

Reply:
ConcurrentQueue cannot be satisfied:

1. It is necessary to follow the `MessageConfig` based on the `Message.Key`, this must be done using Dictionary like object.
2. It needs to be able to delete the specified MessageConfig, But the operation of ConcurrentQueue can only be done at the head or tail of the queue, and cannot be done from the middle.